zerver/models.py

```diff
@@ -364,7 +364,16 @@ class Realm(models.Model):
     allow_edit_history: bool = models.BooleanField(default=True)
 
     DEFAULT_COMMUNITY_TOPIC_EDITING_LIMIT_SECONDS = 259200
-    allow_community_topic_editing: bool = models.BooleanField(default=True)
+
+    EDIT_TOPIC_POLICY_EVERYONE = 1
+    EDIT_TOPIC_POLICY_ADMINS_ONLY = 5
```

Comment:
Have used these values here and kept `2,3,4` values are kept for other roles to keep the values according to hierarchy of role.

Reply:
Can we rather than using 1/5 use the `COMMON_POLICY_TYPES.POLICY_EVERYONE` named values? (total pseudocode here).